src/main/java/org/terasology/climateConditions/FrostbiteSystem.java

```diff
@@ -65,7 +65,7 @@ public void hypothermiaLevelChanged(HypothermiaLevelChangedEvent event, EntityRe
         //Frostbite Effect remains active for Hypothermia Levels 3 and greater.
         if(newLevel == 3 && oldLevel < newLevel) {
             applyFrostbite(player);
-        } else if(oldLevel == 3 && newLevel > oldLevel) {
+        } else if(oldLevel == 3 && newLevel < oldLevel) {
```

Comment:
Maybe let's switch the level variables so that it's aligned with the one above - guess that's easier to see the difference.
```suggestion
        } else if (oldLevel == 3 && oldLevel > newLevel) {
```